test/src/test/java/hudson/PluginManagerInstalledGUITest.java

```diff
@@ -60,8 +61,10 @@ public PluginManager getPluginManager() {
                         try {
                             return super.loadBundledPlugins();
                         } finally {
-                            copyBundledPlugin(PluginManagerInstalledGUITest.class.getResource("/WEB-INF/detached-plugins/cvs.hpi"), "cvs.jpi"); // cannot use installDetachedPlugin at this point
-                            copyBundledPlugin(PluginManagerInstalledGUITest.class.getResource("/plugins/tasks.jpi"), "tasks.jpi");
```

Comment:
Original test case assumed that the implicit `tasks` → detached `cvs` dep could be used to test everything else. Was not rich enough data.